src/App.tsx

```diff
@@ -52,6 +62,12 @@ export default function App() {
 
       <div>
 
+        <div>
+          <h1>Carrossel de fotos</h1>
```

Comment:
poderia usar o componente H1 seu
```suggestion
          <H1>Carrossel de fotos</H1>
```